.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules/
 build/
-app-template/src/config/**
\ No newline at end of file
+app-template/src/config/**
+undoLink.sh
```

Comment:
add these to the readme!

Reply:
and check them in (if they are a useful dev tool)